packages/react-is/src/ReactIs.js

```diff
@@ -90,6 +90,23 @@ export function isForwardRef(object: any) {
 export function isFragment(object: any) {
   return typeOf(object) === REACT_FRAGMENT_TYPE;
 }
+export function isNode(object: any) {
+  switch (typeof object) {
+    case 'number':
+    case 'string':
+    case 'undefined':
+      return true;
+    case 'boolean':
+      return !object;
+    case 'object':
+      if (Array.isArray(object)) {
+        return object.every(isNode);
+      }
+      return object === null || isElement(object);
+    default:
+      return false;
+  }
```

Comment:
Is it worth calling out Symbol in some way?

Reply:
seems like no? you really can't render them right?

Reply:
Ah... right! Returning false as a default is probably sufficient.